### ISSUE TITLE
Loinc 13834-7:  "Total IgE [Units/volume] in Serum by Radioallergosorbent test (RAST)"

### ISSUE THREAD

**Issue by kingmanzhang** [obophenotype/human-phenotype-ontology#4188]
Suggest creating a new term for Loinc: 13834-7 [Total IgE [Units/volume] in Serum by Radioallergosorbent test (RAST)] 
New term label: Abnormal IgE level
New term comment (if any): We have *IgE deficiency* and  *Increase IgE level* but not their parent
Your biocurator ID for loinc2hpo (if desired): JGM:azhang

**Comment by pnrobinson** [comment]
Probably we need to add terms for all of the categories here
https://hpo.jax.org/app/browse/term/HP:0010702
